docs/flatcar-linux/kvm-libvirt.md

```diff
@@ -78,7 +78,7 @@ Terraform v0.11.13
 ```
 
 Add the [terraform-provider-ct](https://github.com/poseidon/terraform-provider-ct) plugin binary for your system
-to `~/.terraform.d/plugins/`, noting the `_v0.3.1` suffix.
+to `~/.terraform.d/plugins/`, noting the `_v0.4.0` suffix.
```

Comment:
Was the ct provider wrongly specified before? Or is this needed to update to terraform 0.12?

It seems 0.3.2 added support for terraform 0.12 (https://github.com/poseidon/terraform-provider-ct/releases), so maybe this was fine just not supported terraform 0.12?

Maybe a commit title like:

```
docs/kvm-libvirt: Update instructions to use ct 0.4.0

Next commits will migrate this to terraform 0.12 and the ct provider should be updated to support terraform 0.12 too.
```

Does it make sense?

Reply:
Please notice that this commit just fixes a typo on the documentation. ct 0.4.0 was already used.

Reply:
Ohh, cool. Didn't notice!